README.md

```diff
@@ -223,6 +223,11 @@ values).
 | ------------------------------------------------------------- | -------------- |
 | [`allowJs`](https://www.typescriptlang.org/tsconfig/#allowJs) | `false`        |
 
+### Troubleshooting
+
+Possible errors that can arise from not having `allowJs: false`:
+- `TypeError [ERR_INVALID_URL_SCHEME]: The URL must be of scheme file` from tsx when trying to run `npx local-action`.
```

Comment:
```suggestion
Possible errors that can arise from not having `allowJs: false`:

- `TypeError [ERR_INVALID_URL_SCHEME]: The URL must be of scheme file` from tsx when trying to run `npx local-action`.
```

Small tweak because markdownlint will not like the lack of space between the paragraph and list.